package.json

```diff
@@ -1,78 +1,20 @@
 {
-  "name": "prosemirror-jdita",
-  "version": "0.0.1",
-  "description": "Use JDita in prosemirror",
-  "main": "./lib/index.js",
-  "types": "./lib/index.d.ts",
+  "name": "prosemirror-jdita-package",
+  "version": "0.1.1",
+  "description": "Prosemirror plugin to edit JDita documents",
+  "main": "index.js",
+  "repository": "https://github.com/ccheraa/prosemirror-jdita.git",
```

Comment:
This needs some fixing from the left-version. Can you please set:

```json
"repository": {
    "type": "git",
    "url": "git+https://github.com/evolvedbinary/prosemirror-jdita.git"
  },
  "keywords": [	
    "LwDITA",	
    "DITA",	
    "XML",	
    "JSON"	
  ],	
  "author": "Evolved Binary",	
  "license": "AGPL-3.0",	
  "bugs": {	
    "url": "https://github.com/evolvedbinary/prosemirror-jdita/issues"	
  },	
  "homepage": "https://github.com/evolvedbinary/prosemirror-jdita#readme"
```

Reply:
Oh yes, I forgot about it.
It should be changed in the other 2 `package.json` files for the 2 workspaces